yarpl/include/yarpl/flowable/Flowable.h

```diff
@@ -120,41 +120,41 @@ template <typename T>
 template <typename Function, typename R>
 Reference<Flowable<R>> Flowable<T>::map(Function function) {
   return make_ref<MapOperator<T, R, Function>>(
-      get_ref(this), std::move(function));
+      this->ref_from_this(this), std::move(function));
 }
 
 template <typename T>
 template <typename Function>
 Reference<Flowable<T>> Flowable<T>::filter(Function function) {
   return make_ref<FilterOperator<T, Function>>(
-      get_ref(this), std::move(function));
+      this->ref_from_this(this), std::move(function));
```

Comment:
just curious, is the "this->" necessary?

Reply:
It's not; it'd only be required if `enable_get_ref` was templated. All instances being prefixed with `this->` are a holdover from a version which did have a templated `enable_get_ref`.